src/Pollers/DeviceMappingPoller.php

```diff
@@ -154,13 +157,26 @@ public function poll(array $work):array
      */
     private function getDeviceMapper(array $hostWithDeviceType, Device $device)
     {
+       
         switch ($hostWithDeviceType['type_query_result'])
         {
             case "1.3.6.1.4.1.161.19.250.256":
                 $mapper = new CambiumCanopyPMPAccessPointMapper($device);
                 break;
-            case "1.3.6.1.4.1.17713.21":
-            case "1.3.6.1.4.1.17713.21.1.1.2":
+	    case "1.3.6.1.4.1.17713.21":
+	    case "1.3.6.1.4.1.17713.21.9.37":
+		$mapper = new CambiumEpmpAccessPointMapper($device); //EPMP3000 (FCC)
+		break;
+	    case "1.3.6.1.4.1.17713.21.9.40":
+                $mapper = new CambiumEpmpAccessPointMapper($device); //EPMP3000 (ROW/ETSI)
+                break;
+	    case "1.3.6.1.4.1.17713.21.9.43":
+                $mapper = new CambiumEpmpAccessPointMapper($device); //EPMP3000L (FCC)
+                break;
+	    case "1.3.6.1.4.1.17713.21.9.44":
+                $mapper = new CambiumEpmpAccessPointMapper($device); //EPMP3000L (ROW/ETSI)
+                break;
+	    case "1.3.6.1.4.1.17713.21.1.1.2":
```

Comment:
Need to fix the indenting here to match the surrounding case statements. You can also just case the same thing 4 times with a single break rather than repeating this.

Reply:
The changes have been committed.. I think. ?